pkg/eval/mods/bundled/epm.elv.go

```diff
@@ -109,7 +109,14 @@ debug-mode = $false
 ]
 
 # Internal configuration
--data-dir = ~/.elvish
+-data-dir = (
+  if (has-env XDG_CONFIG_HOME) {
+    put $E:XDG_CONFIG_HOME/elvish
```

Comment:
An empty XDG_CONFIG_HOME should be treated as if it is unset; the condition here should be

`(not (eq $XDG_CONFIG_HOME ''))`

Reply:
Ah, and the design is to use XDG_DATA_HOME, not XDG_CONFIG_HOME: https://github.com/elves/elvish/issues/383#issuecomment-869217648

Reply:
Oops. Sorry about that. Fixed it.